index.js

```diff
@@ -1,11 +1,30 @@
 var path = require('path');
+var fs = require('fs');
 var findParentDir = require('find-parent-dir');
 
+/**
+ * Resolves the url in the top level node_module directory
+ * @param {String} targetUrl url to resolve
+ * @param {String} source file or directory which is the base point resolving targetUrl
+ */
+function resolve(targetUrl, source) {
+  var packageRoot = findParentDir.sync(source, 'node_modules');
+
+  if (!packageRoot) {
+    return null; // file could not be resolved as npm package
```

Comment:
Remove comment. Side note: its also incorrect - we aren't looking for files here, just parent directories that have `node_module` folders in them.